### DISCUSSION TITLE
Unable to Run UNO Application on Uno Mobile Platform Using VS Code

### DISCUSSION THREAD

**Discussion by rakeshghumatkar** [unoplatform/uno#13548] [Q&A]
Hello, I'm trying to run a UNO application using the UNO Mobile Platform in Visual Studio Code. However, when I click the button, nothing happens. Interestingly, the application works flawlessly in WebAssembly and Skia.GTK. I've also installed an emulator extension for mobile debugging. The project builds successfully without any errors. I encounter the same issue when creating a new UNO application. Does anyone know why this is happening and how to resolve the issue?

**Comment by jeromelaban** [comment]
Thanks for the report! Which mobile target do you use? When you want to debug, there's an output tab that shows up, could you share what's in it?
